### PR TITLE
Update 02-Testing_for_Default_Credentials.md

### PR DESCRIPTION
**Pull request by omranisecurity**
Add tools and references

### CODE REVIEW

**Review by kingthorin — COMMENTED** [review]
Please follow the bot's advice.

File: latest/4-Web_Application_Security_Testing/04-Authentication_Testing/02-Testing_for_Default_Credentials.md [hunk 1]
```diff
@@ -75,9 +75,11 @@ This type of issue of often difficult to identify from a black-box perspective.
...
 - [THC Hydra](https://github.com/vanhauser-thc/thc-hydra)
 - [Nikto 2](https://www.cirt.net/nikto2)
+- [Nuclei](https://github.com/projectdiscovery/nuclei)
 
 ## References
 
 - [CIRT](https://cirt.net/passwords)
 - [SecLists Default Passwords](https://github.com/danielmiessler/SecLists/tree/master/Passwords/Default-Credentials)
 - [DefaultCreds-cheat-sheet](https://github.com/ihebski/DefaultCreds-cheat-sheet/blob/main/DefaultCreds-Cheat-Sheet.csv)
+- [Nuclei-Templates](https://github.com/projectdiscovery/nuclei-templates/tree/6b26c63d8f63b2a812a478f14c4c098b485d54b4/http/default-logins)
```
Comment (kingthorin) [inline latest/4-Web_Application_Security_Testing/04-Authentication_Testing/02-Testing_for_Default_Credentials.md:85]: This does not seem to be reference material.

Comment (omranisecurity) [reply latest/4-Web_Application_Security_Testing/04-Authentication_Testing/02-Testing_for_Default_Credentials.md:85]: Yes, that's correct. Is the Nuclei Tool acceptable?

Comment (kingthorin) [reply latest/4-Web_Application_Security_Testing/04-Authentication_Testing/02-Testing_for_Default_Credentials.md:85]: Yup the tool is good, and you could even add the template repo as a secondary tool entry. (Like Nuclei as a bull, then templates as a sub-bullet)

**Pull request closed without merging**